src/borg/helpers.py

```diff
@@ -1863,10 +1890,10 @@ def hash_item(self, hash_function, item):
         return hash.hexdigest()
 
     def format_time(self, key, item):
-        return format_time(safe_timestamp(item.get(key) or item.mtime))
+        return OutputTimestamp(safe_timestamp(item.get(key) or item.mtime), json_microseconds=True)
```

Comment:
so, if we want usecs here in json, maybe simpler to just always/everywhere use usecs for json?